### PR TITLE
Web-Descriptions —> Web

### PR DESCRIPTION
**Pull request by marianbitsov23**
### Description Feature
  - Added description input in the User info page & Edit personal info page
  - Added description display on the User Card
  - Added Landing page
  - Bug fixes

### CODE REVIEW

**Review by msimov02 — APPROVED** [review]
LGTM! Nice Landing Page.

File: Web/tues_pairs/src/components/Pages/Landing/landing.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import * as ROUTES from '../../../constants/routes';
+import GitHubIcon from '@material-ui/icons/GitHub';
+
+class LandingPageBase extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+
+        }
```
Comment (GGLozanov) [inline Web/tues_pairs/src/components/Pages/Landing/landing.jsx]: ?

**Review by GGLozanov — APPROVED** [review]
Alright, I suppose. Just answer the questions I had. Otherwise, good job.

File: Web/tues_pairs/src/components/Pages/Landing/landing.jsx (hunk 1 above)
Comment (marianbitsov23) [reply Web/tues_pairs/src/components/Pages/Landing/landing.jsx]: Oops, I missed on that.

**Pull request merged**